docs/markdown/Using-wraptool.md

```diff
@@ -96,3 +96,16 @@ but available in WrapDB will automatically be downloaded.
 
 Automatic fetch of WrapDB subprojects can be disabled by removing the file
 `subprojects/wrapdb.json`, or by using `--wrap-mode=nodownload`.
+
+## Self-hosted Wrap database
+
+If for whatever reason you want to use self-hosted or proxied Wrap database *(since 1.X.X)* you may set mirror address to be used for the workspace:
+
+```console
+$ meson wrap set-db-mirror user:password@wrapdb.mydomain.invalid:8080
```

Comment:
I really don't like that this does not allow to set a protocol. For now the wrap database speaks only https, but I can imagine a future where a different protocol is implemented.

Reply:
Like ftp, sftp, git? This will require parsing urls differently. An ssh/git user might want to pass a key somehow. Doable, but not sure if I want to open this can of worms yet.

Having a `file:` or plain `http:` protocol will be convenient for local testing, but not really useful otherwise.

Reply:
I don't understand what you mean with "opining this can of worms". You are making the location of the wrap database configurable. What I am asking is to do not assume that the protocol is https and make the protocol part of the URL part of the configuration string instead of hard-coding it to be https. Where are the worms in this request?

Reply:
Wish granted. There is enforcement of `https` in the code as if the only other alternative is http. I didn't get it right yet. WIP.